examples/bri-3/src/bri/workgroup/workgroups/models/workgroupComponent.interface.ts

```diff
@@ -4,17 +4,20 @@ import { Agreement } from '../../../storage/models/agreement';
 import { Invitation } from '../../invitations/models/invitation';
 
 export interface IWorkgroupComponent {
+
     workgroups: Workgroup[];
     invitations: Invitation[];
     archivedWorkgroups: Workgroup[];
-    getWorkgroups(): Workgroup[];
+
+    getWorkgroups(id?: string[]): Workgroup[];
```

Comment:
consitent naming with above methods that accept optional array. Either getAllWorkgroups or getParticipants